linux_sql/.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
+/shelf/
```

Comment:
`git rm linux_sql/.idea`
do not commit .idea folder

Reply:
make sure you review all changes files for future PRs
https://github.com/jarviscanada/jarvis_data_eng_HabibMohamed/pull/1/files